File: 7_项目搭建/android/IDEA/IFengBigData/src/main/java/cc/m2u/ifengbigdata/activity/MainActivity.java
package cc.m2u.ifengbigdata.activity;

import android.os.Bundle;
import android.view.MenuItem;
import cc.m2u.ifengbigdata.R;
import cc.m2u.ifengbigdata.activity.base.BaseCompatActivity;

/**
 * 主页
 */
public class MainActivity extends BaseCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
    }

    @Override
    protected int getLayoutId() {
        return R.layout.activity_main;
    }

    @Override
    protected int getMenuId() {
        return 0;
    }

    @Override
    public void whenOptionsItemSelected(MenuItem item) {

    }
}

File: 7_lottery/src/main/java/cc/m2u/lottery/activity/test/MainActivity.kt
package cc.m2u.lottery.activity.test

/**
 * 主页呗
 * Created by wx on 2017/6/1.
 */
class MainActivity : cc.m2u.lottery.activity.base.BaseCompatActivity() {
    override fun getLayoutId(): Int {
        return cc.m2u.lottery.R.layout.activity_main
    }

    override fun getMenuId(): Int {
        //return 0
        return cc.m2u.lottery.R.menu.main
    }

    override fun whenOptionsItemSelected(item: android.view.MenuItem?) {
        if (item!!.itemId == cc.m2u.lottery.R.id.add_new_todo) {
            //startAddNewActivty();
            android.widget.Toast.makeText(this, "新建", android.widget.Toast.LENGTH_SHORT).show()
        }
    }

    override fun onCreate(savedInstanceState: android.os.Bundle?) {
        super.onCreate(savedInstanceState)
        //
        h.sendEmptyMessageDelayed(cc.m2u.lottery.activity.test.MainActivity.Companion.MSG_GO, 1000);
        toolbarTitle.text = "MainActivityKt!"
    }

    companion object {
        private val MSG_GO: Int = 1;
    }

    private val h: android.os.Handler = object : android.os.Handler() {
        override fun handleMessage(msg: android.os.Message?) {
            when (msg!!.what) {
                cc.m2u.lottery.activity.test.MainActivity.Companion.MSG_GO -> { //startToMain2();
                    var intent: android.content.Intent = android.content.Intent(this@MainActivity, Main2Activity::class.java);
                    startActivity(intent);
                }
                else -> {
                }
            }
            super.handleMessage(msg)
        }
    }

}